libtsuba/src/RDG.cpp

```diff
@@ -43,7 +43,7 @@ namespace {
 // special partition property names
 const char* kMirrorNodesPropName = "mirror_nodes";
 const char* kMasterNodesPropName = "master_nodes";
-const char* kLocalToTGlobalPropName = "local_to_global_vector";
+const char* kLocalToGlobalIDPropName = "local_to_global_id";
 
```

Comment:
Is this change backward compatible with previous RDGs?

Reply:
No, it isn't. Is that an issue? Do you want me to add support for backward compatibility?

Reply:
@tylershunt has been fastidious about maintaining backward compatibility for storage, which means most people remain blissfully ignorant of our goings-on.  I have a bit more of a rip the band-aid off approach but bear in mind that if you change the on-disk format, PayPal and our input testing and random RDGs on S3 will not load.

Reply:
Thanks for pointing this out. I will create another PR to be backward compatible.